### ISSUE TITLE
Rare failures on the tutorial example

### ISSUE THREAD

**Issue by dandv** [xolvio/chimp#56]
Posted at https://chimp.readme.io/v1.0/discuss/55c75be337d9670d00ca4485 but not sure if it's a tutorial oversight, or if there's some weird bug with Chimp.

I've had very odd results with this test after changing the waitForExist line to

    browser.waitForExist('a=doesntreallyexist');

If I rename the `step_defs.js` file and/or move it under some subdirectory and/or change `doesntreallyexist` to some other nonexistent string` a bunch of times (couldn't find a clearly reproducible sequence), the test will correctly fail. Most of the time the test will incorrectly pass.

Here's a log excerpt:

```
[chimp] Running...

Feature: Search the Web

  As a human
  I want to search the web
  So I can find information

  @watch
  Scenario: Search for Xolv.io   # features\search.feature:8
    Given I have visited Google  # features\search.feature:9
    When I search for "Xolv.io"  # features\search.feature:10
    Then I see "Xolv.io"         # features\search.feature:11


1 scenario (1 passed)
3 steps (3 passed)
[chimp] Running...
[chimp] Running...

Feature: Search the Web

  As a human
  I want to search the web
  So I can find information

  @watch
  Scenario: Search for Xolv.io   # features\search.feature:8
    Given I have visited Google  # features\search.feature:9
    When I search for "Xolv.io"  # features\search.feature:10
    Then I see "Xolv.io"         # features\search.feature:11


1 scenario (1 passed)
3 steps (3 passed)
[chimp] Running...

Feature: Search the Web

  As a human
  I want to search the web
  So I can find information

  @watch
  Scenario: Search for Xolv.io   # features\search.feature:8
    Given I have visited Google  # features\search.feature:9
    When I search for "Xolv.io"  # features\search.feature:10
    Then I see "Xolv.io"         # features\search.feature:11
      Error: element (a=pp) still not existing after 10000ms
        at World.<anonymous> (features\search.steps.js:13:13)

          - - - - -




(::) failed steps (::)

Error: element (a=pp) still not existing after 10000ms
  at World.<anonymous> (features\search.steps.js:13:13)

    - - - - -


Failing scenarios:
features\search.feature:8 # Scenario: Search for Xolv.io

1 scenario (1 failed)
3 steps (1 failed, 2 passed)

[chimp] Finished running async processes with errors
[chimp] run complete with errors Cucumber steps failed
[chimp] Running...

Feature: Search the Web

  As a human
  I want to search the web
  So I can find information

  @watch
  Scenario: Search for Xolv.io   # features\search.feature:8
    Given I have visited Google  # features\search.feature:9
    When I search for "Xolv.io"  # features\search.feature:10
    Then I see "Xolv.io"         # features\search.feature:11
      Error: element (a=ppihiu) still not existing after 10000ms
        at World.<anonymous> (features\search.steps.js:13:13)

          - - - - -




(::) failed steps (::)

Error: element (a=ppihiu) still not existing after 10000ms
  at World.<anonymous> (features\search.steps.js:13:13)

    - - - - -


Failing scenarios:
features\search.feature:8 # Scenario: Search for Xolv.io

1 scenario (1 failed)
3 steps (1 failed, 2 passed)

[chimp] Finished running async processes with errors
[chimp] run complete with errors Cucumber steps failed
[chimp] Running...
[chimp] Running...

Feature: Search the Web

  As a human
  I want to search the web
  So I can find information

  @watch
  Scenario: Search for Xolv.io   # features\search.feature:8
    Given I have visited Google  # features\search.feature:9
    When I search for "Xolv.io"  # features\search.feature:10
    Then I see "Xolv.io"         # features\search.feature:11


1 scenario (1 passed)
3 steps (3 passed)
```

I'm running `chimp --watch` on Windows 7 with Chrome. Hopefully I'm missing something obvious.

**Comment by Sanjo** [comment]
I think this is fixed with 0.14.0

**Comment by dandv** [comment]
Confirm fixed. Thanks for the quick response!
